.circleci/config.yml

```diff
@@ -0,0 +1,295 @@
+version: 2.1
+
+
+orbs:
+  mim:
+    commands:
+      install_erlang:
+        parameters:
+          otp:
+            type: string
+            description: Version of the Erlang package to install
+        steps:
+        - run:
+            name: Install Erlang
+            command: |
+              wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
+              sudo dpkg -i erlang-solutions_1.0_all.deb
+              sudo apt-get update
+              sudo apt-get install -y esl-erlang=<<parameters.otp>>
+      fetch_packages:
+        steps:
+        - run:
+            name: Install basic packages
+            command: |
+              sudo apt install unixodbc-dev
+              sudo apt install unixodbc
+              sudo apt install tdsodbc
+    jobs:
+      build_and_small_tests:
+        parallelism: 1
+        machine:
+          docker_layer_caching: false
```

Comment:
Do we need to set this? We don't really use docker server anyway, so this seems to be redundant.
On the other hand, we don't specify OS version, so we rely on default value, which may change without warning and possibly break the build.

Reply:
Btw. we can define an executor in top section of the file to only define this once:
```
executors:
  awesome-executor:
    machine:
      - image: ubuntu-1604:201903-01`
```
https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21

Reply:
Thats true, I will put exact machine version.
` docker_layer_caching` is just leftover as it was set to true before. Removing